litmus-portal/frontend/src/pages/AnalyticsPage/index.tsx

```diff
@@ -1,15 +1,68 @@
+/* eslint-disable no-unused-expressions */
 import { useQuery } from '@apollo/client';
 import { Typography } from '@material-ui/core';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector } from 'react-redux';
 import { useLocation } from 'react-router-dom';
+import moment from 'moment';
 import Loader from '../../components/Loader';
 import Scaffold from '../../containers/layouts/Scaffold';
 import { WORKFLOW_DETAILS } from '../../graphql';
-import { Workflow, WorkflowDataVars } from '../../models/graphql/workflowData';
+import {
+  ExecutionData,
+  Workflow,
+  WorkflowDataVars,
+} from '../../models/graphql/workflowData';
 import { RootState } from '../../redux/reducers';
+import WorkflowRunsBarChart from '../../views/ChaosWorkflows/BrowseAnalytics/WorkflowRunsBarChart';
+import PopOver from '../../views/ChaosWorkflows/BrowseAnalytics/PopOver';
+import useStyles from './styles';
+
+interface WorkflowRunData {
+  testsPassed: number;
+  testsFailed: number;
+  resilienceScore: number;
+  testDate: string;
+  workflowRunID: string;
+}
+
+interface SelectedWorkflowRunData {
+  testsPassed: number;
+  testsFailed: number;
+  resilienceScore: number;
+  testDate: string;
+  xLoc: number;
+  yLoc: number;
+  workflowRunID: string;
+}
 
 const AnalyticsPage: React.FC = () => {
+  const classes = useStyles();
+
+  const [selectedWorkflowRunData, setSelectedWorkflowRunData] = React.useState<
+    SelectedWorkflowRunData
+  >({
+    testsPassed: 0,
+    testsFailed: 0,
+    resilienceScore: 0,
+    testDate: '',
+    xLoc: 0,
+    yLoc: 0,
+    workflowRunID: '',
+  });
+
+  const workflowID: string = 'hello';
```

Comment:
Why do you have a dummy 'hello' string as the workflowID!

Reply:
Thats not required will be removing it @S-ayanide